easybuild/easyconfigs/l/LAMMPS/LAMMPS-patch_4Feb2020-intel-2019a-Python-3.7.2-kokkos.eb

```diff
@@ -0,0 +1,115 @@
+name = 'LAMMPS'
+version = 'patch_4Feb2020'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-kokkos'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
```

Comment:
@Darkless012 Can we bump this to `intel/2019b` (and use a more sensible version); see also remarks in #9884